schemas/context/language.schema.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
This something that should be part of common - since if we are going to standardize this, it should be across the board for all schemas.